alexa.py

```diff
@@ -37,6 +33,8 @@
 # Timestamp based verification shouldn't be used in production. Use at own risk
 # app.config['ASK_VERIFY_TIMESTAMP_DEBUG'] = True
 
+# https://github.com/johnwheeler/flask-ask/issues/43
+app.config['ASK_VERIFY_REQUESTS'] = False
 
```

Comment:
I personally don't want to disable this, so at the very least, it should be configurable.